Pipfile

```diff
@@ -14,6 +14,8 @@ matplotlib = "*"
 scipy = "*"
 jupyter = "*"
 sklearn = "*"
+pandas = "*"
+plotly = "*"
```

Comment:
@jia200x @gwenzel here we need to make a decision whether we make plotly a dependency of opensir, or we write in the notebook that plotly is required and the user has to install it using pip install plotly in order to run the notebook.

At the moment, I wouldn't be against of making plotly a dependency as the plotly figures created by @gwenzel may constitute in the future a function of model.plot().

Reply:
> @jia200x @gwenzel here we need to make a decision whether we make plotly a dependency of opensir, or we write in the notebook that plotly is required and the user has to install it using pip install plotly in order to run the notebook.

As described above, the user of this Jupyter is the End User, not the developer user.
There are a lot of problems of using Pipenv for end users (e.g if the user doesn't have Python 3.7, it won't be able to run opensir).

> At the moment, I wouldn't be against of making plotly a dependency as the plotly figures created by @gwenzel may constitute in the future a function of model.plot().

If the model.plot uses plotly, then that PR should add the dependency. Adding spare dependencies is not recommended because it adds more "moving parts" if not used